cbind/libp2p_thread/inter_thread_communication/requests/libp2p_lifecycle_requests.nim

```diff
@@ -50,7 +50,7 @@ proc destroyShared(self: ptr LifecycleRequest) =
 
 proc process*(
     self: ptr LifecycleRequest, libp2p: ptr LibP2P
-): Future[Result[string, string]] {.async.} =
+): Future[Result[Opt[string], string]] {.async.} =
```

Comment:
i'am not sure about this. code would be better without this change. it makes things complicated and returning `""` is essentially  `Opt.none(string)`.